src/js/components/Widgets/ReadMore.jsx

```diff
@@ -34,7 +34,7 @@ export default class ReadMore extends Component {
           link_text = "More";
         }
         if (collapse_text === undefined) {
-          collapse_text = " ...Less";
+          collapse_text = " ...Show Less  ";
```

Comment:
Do we even need this 'Show less' toggle? I like displaying a truncated version when appropriate, but once someone clicks to show more, is it likely to add that much more clutter (length) to the page that folks will want to collapse it again? If it is a more common case that these statements will be so long people will want to toggle them 'open and closed' maybe we should consider alternative patterns. ?